packages/config/src/options.ts

```diff
@@ -198,6 +198,12 @@ const driverConfigOptions: Array<DriverConfigOption> = [
     defaultValue: 60000,
     validation: validate.isNumber,
     overrideLevel: 'any',
+  }, {
+    name: 'experimentalCspAllowList',
+    defaultValue: false,
+    validation: validate.validateAny(validate.isBoolean, validate.isSubsetOf('script-src-elem', 'script-src', 'default-src', 'sandbox', 'form-action', 'navigate-to')),
+    overrideLevel: 'any',
```

Comment:
We dont want to support override this at test time,
```suggestion
    overrideLevel: 'never',
```

Reply:
You are correct...although I'd like to at some point. 😉